### PR TITLE
Amount should be more than MinSelfDelegation.

### PR DESCRIPTION
**Pull request by zouguangxian**

### CODE REVIEW

**Comment by LeoHChen** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by gupadhyaya** [comment]
@zouguangxian no need for this check as it is done in `if err := wrapper.SanityCheck(maxBLSKeyAllowed); err != nil`

File: core/staking_verifier.go [hunk 1]
```diff
@@ -87,6 +87,9 @@ func VerifyAndCreateValidatorFromMsg(
 	if msg.Amount.Sign() == -1 {
 		return nil, errNegativeAmount
 	}
+	if msg.Amount.Cmp(msg.MinSelfDelegation) == -1 {
+		return nil, errInsufficientBalanceForStake
```
Comment (rlan35) [inline core/staking_verifier.go:91]: Should use this error: ErrInvalidSelfDelegation

Comment (rlan35) [reply core/staking_verifier.go:91]: And also true for @gupadhyaya 's comment.

**Comment by gupadhyaya** [comment]
no longer required, hence closing.

**Pull request closed without merging**